build.rs

```diff
@@ -1,5 +1,25 @@
 extern crate pkg_config;
 
 fn main() {
-    pkg_config::find_library("alsa").unwrap();
+    let missing: String = String::from("\"pkg-config\" \"--libs\" \"--cflags\" \"alsa\"");
+
+    match pkg_config::probe_library("alsa") {
+        Ok(_) => {},
+        Err(e) => {
+            match e {
+                pkg_config::Error::Failure {command, output} => {
+                    if command == missing {
+                        panic! (
+                            "Could not find the alsa library. Have you installed the library on your system?\n\n\
+                            For Fedora users:\n# dnf install alsa-lib-devel\n\n\
+                            For Debian/Ubuntu users:\n# apt-get install libasound2\n\n"
+                        );
+                    } else {
```

Comment:
pkg-config could potentially fail for other reasons than that the library is not installed - I would prefer not to discard the original message here

Reply:
Hello @diwic 
Thanks for your quick response! It might looks like I discard the original message, but the output of `e` is basically the original message. I think this is the most generic way of handling the other variants because it is likely that [more variants will come](https://docs.rs/pkg-config/0.3.11/src/pkg_config/lib.rs.html#137-139) with probably different types.

Reply:
So I mean in the case of `pkg_config::Error::Failure` + `command == missing` you *do* discard the original message.